src/content/en/2021/jamstack.md

```diff
@@ -1,20 +1,521 @@
+
+Jekyll is a stand out, achieving a score of almost 70 which is a great result for such a mastodon in the SSG area. Learn more about <a hreflang="en" href="https://web.dev/lighthouse-performance/">Lighthouse performance audit</a> to understand exactly what measures are included in this score.
+
+#### Accessibility score
+
+Lighthouse also runs <a hreflang="en" href="https://web.dev/lighthouse-accessibility/">audits to measure accessibility</a> and here we seem to have better results:
+
+{{ figure_markup(
+  image="median-lighthouse-accessibility-score.png",
+  caption="Median Lighthouse accessibility score",
+  description="Bar chart represents median Lighthouse accessibility score.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRUiAdcUCSYyS7cQcQlE9uKeMSacgRudtf4pFRkIKKCt8Nw18qOod6TaAUL7tfm8pUbCPKShEN0jJOS/pubchart?oid=1270511864&format=interactive",
+  sheets_gid="745310638",
+  sql_file="median_lighthouse_score.sql"
+) }}
+
+There are limits to what can be checked in an automated accessibility check, but this is still positive. Read the [Accessibility](./accessibility) chapter for more on this subject.
+
+#### SEO score
+
+{{ figure_markup(
+  image="median-lighthouse-seo-score.png",
+  caption="Median Lighthouse SEO score",
+  description="Bar chart represents median Lighthouse SEO score. All Jamstack sites provide great SEO scores. From 90 to 92.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRUiAdcUCSYyS7cQcQlE9uKeMSacgRudtf4pFRkIKKCt8Nw18qOod6TaAUL7tfm8pUbCPKShEN0jJOS/pubchart?oid=813819935&format=interactive",
+  sheets_gid="745310638",
+  sql_file="median_lighthouse_score.sql"
+) }}
+
+Simialrly, all Jamstack sites provide great SEO scores from 90 to 92.
+
+{# TODO Any thoughts on why? Is this something SSGs concentrate on more than Performance or Accessibility? #}
+
+#### Best Practices score
+
+<a hreflang="en" href="https://web.dev/lighthouse-best-practices/">Lighthouse Best Practices</a> category makes sure that site avoids using unhealthy common patterns which dramatically changes User Experience, for example use of the outdated `document.write()`.
+
+{{ figure_markup(
+  image="median-lighthouse-best-practices-score.png",
+  caption="Median Lighthouse best practices score",
+  description="Bar chart represents median Lighthouse best practices score.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRUiAdcUCSYyS7cQcQlE9uKeMSacgRudtf4pFRkIKKCt8Nw18qOod6TaAUL7tfm8pUbCPKShEN0jJOS/pubchart?oid=343006289&format=interactive",
+  sheets_gid="745310638",
+  sql_file="median_lighthouse_score.sql"
+) }}
+
+All sites results mostly 87, and 93 for for some of them.
+
+{# TODO not sure this is a great stat to include since it's such and odd bucket of other audits. What is the reader supposed to take away from this? Maybe should consider dropping? #}
```

Comment:
Shoulld we drop the Best Practice section @denar90 ? Is it really telling us much?

Reply:
dropped